bridges/snowbridge/pallets/system-v2/src/lib.rs

```diff
@@ -113,6 +130,10 @@ pub mod pallet {
 		InvalidUpgradeParameters,
 	}
 
+	/// Relayer reward tips that were added but the nonce was already consumed.
```

Comment:
We should flesh this comment out, describing why we want to keep track of these lost tips (so we can implement reclaiming in a future version. Also "nonce was already consumed." is a bit too specific. Can just say like if the tip could not be processed for any reason.

Reply:
Resolved in https://github.com/vgeddes/polkadot-sdk/commit/a0fda22f8e4a362f46b101909969750fd2cd6ac7

Reply:
Can't find the comment in this file, seems review outdated code?